client/indexer-service/src/handler.rs

```diff
@@ -445,6 +453,12 @@ impl<Runtime: StorageEnableRuntime> IndexerService<Runtime> {
                 // Convert EVM tx hash to bytes if present
                 let tx_hash_bytes = evm_tx_hash.map(|h| h.as_bytes().to_vec());
 
+                // Check if this file key is already present in the bucket of the MSP
+                // This is because the `MutationsApplied` event won't be emitted for this file key when
+                // the MSP accepts it because the MSP is already storing it.
```

Comment:
Add something like: 
This would happen when the first SR was accepted and the user opens a new one to increase redundancy